src/briefcase/platforms/linux/appimage.py

```diff
@@ -120,21 +125,48 @@ def verify_host(self):
 
 
 class LinuxAppImageCreateCommand(
-    LinuxAppImageMixin, LocalRequirementsMixin, CreateCommand
+    LinuxAppImageMixin,
+    LocalRequirementsMixin,
+    CreateCommand,
 ):
     description = "Create and populate a Linux AppImage."
 
-    def support_package_filename(self, support_revision):
-        """The query arguments to use in a support package query request."""
-        return f"Python-{self.python_version_tag}-linux-{self.tools.host_arch}-support.b{support_revision}.tar.gz"
+    def finalize_app_config(self, app: AppConfig):
```

Comment:
I also just noticed that this doesn't run if the `create` command is only run implicitly; for instance, as part of `briefcase build`. This seems to be a consequence of `if hasattr(app, "__draft__")` back up in `base.py`.

Reply:
Ah - of course - I need to move the finalise to the mixin location, rather than being defined only in create. Nice catch.